networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/topics/validation/AttestationValidator.java

```diff
@@ -105,7 +106,8 @@ ValidationResult singleOrAggregateAttestationChecks(final Attestation attestatio
     // queue
     // future attestations for processing at the appropriate slot).
     final UnsignedLong currentTimeMillis = secondsToMillis(recentChainData.getStore().getTime());
-    if (isAfterPropagationSlotRange(currentTimeMillis, attestation)) {
+    if (isAfterPropagationSlotRange(currentTimeMillis, attestation)
```

Comment:
Is it just me or should `isAfterPropagationSlotRange` really be `isBeforePropagationSlotRange`?  ie it's valid if it's after the propagation slot range and invalid if it's from too far back.

Reply:
Yes it should be isBefore, I'll do the switch.

Reply:
No, on second thought, `isAfterPropagationSlotRange` is correct. Because it does not refer to the attestation, it refers to the current time. Current time is after the attestations propagation slot range, thus the naming. I'll change these methods to:

- `isCurrentTimeAfterAttestationPropagationSlotRange`
- `isCurrentTimeBeforeMinimumAttestationBroadcastTime`

Reply:
all for 50 character method names.